test/tests/selftest/pbs_managers_operators_ext.py

```diff
@@ -38,7 +38,7 @@
 from tests.selftest import *
```

Comment:
I know that you didn't create this file, but can you please rename it to remove the "ext" at the end ?